styles/navbar.module.css

```diff
@@ -1,20 +1,144 @@
+.starknetIdLogo {
+  cursor: pointer;
+}
+
 .menuItem {
   margin-left: 2.5rem /* 40px */;
   font-size: 0.875rem /* 14px */;
   line-height: 1.25rem /* 20px */;
   text-transform: uppercase;
   cursor: pointer;
-  font-weight: 800;
+  font-weight: 600;
   color: var(--secondary);
+  text-transform: underline;
+  transition: 0.3s ease-in-out;
+  position: relative;
 }
 
-.menuItem {
-  text-transform: underline;
+.menuBurger {
+  margin-left: 2.5rem /* 40px */;
+  cursor: pointer;
+  position: relative;
+}
+
+.menuBurger[aria-expanded="true"]::before,
+.menuBurger:hover::before {
+  content: "";
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  width: 40px;
+  height: 40px;
+  transform: translate(-50%, -50%);
+  background-color: red;
+  z-index: -1;
+  border-radius: 100px;
+  background: var(--light-tertiary);
+}
+
+.menuItem:hover {
+  opacity: 0.7;
 }
 
 .menuItemSmall {
   padding-top: 1rem /* 16px */;
   padding-bottom: 1rem /* 16px */;
   font-size: 0.875rem /* 14px */;
   line-height: 1.25rem /* 20px */;
+  color: var(--dark-content);
+  /* Body/default/bold */
+  font-family: Poppins-Bold;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: bold;
+  line-height: 28px; /* 175% */
+  letter-spacing: 0.16px;
+}
+
+.navbarContainer {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  padding-left: 0.5rem;
```

Comment:
`padding: 0rem 0.5rem` should do here

Reply:
In fact this class isn't used so I am deleting it